Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM alpine:3
+
+ENV SSH_PUBLIC_KEY SSH_PUBLIC_KEY
+
+RUN apk add --no-cache \
+  openssh-server \
+  runuser \
+  git
+
+RUN ssh-keygen -A
```

Comment:
❓ how is this getting stored in `SSH_PUBLIC_KEY`?

Reply:
`SSH_PUBLIC_KEY` is not used while building the image. It's read by the `start.sh` script and written to the authorized keys for the git user. The intent is that the env is passed when running the container. It probably doesn't need to be in the `Dockerfile` at all.

`ssh-keygen -A` generates default key public/private key files. It's meant to be run as root and puts the key files into`/etc/ssh`.